tests/integration/dashboard/test_reviews.py

```diff
@@ -0,0 +1,74 @@
+import pytest
+from django.utils import timezone
+
+from oscar.apps.dashboard.reviews import views
+
+
+now = timezone.now()
+
+
+@pytest.fixture
+def reviewlistview(rf):
+    view = views.ReviewListView()
+    view.request = rf.get('/')
+    return view
+
+
+@pytest.mark.django_db
+class TestReviewListView(object):
```

Comment:
I think we should also be testing whether the queryset/filters that are generated by `get_queryset()` and submethods is actually correct? It ends up being quite a complex query and currently we don't appear to have any tests to check that it is correct.

Reply:
Indeed.

Reply:
@pjstevns would you add suggested test?

Reply:
@sasha0 I've expanded the test-coverage